Index: u-boot/Makefile
===================================================================
--- u-boot.orig/Makefile
+++ u-boot/Makefile
@@ -610,10 +610,10 @@ KBUILD_CFLAGS += $(call cc-option,-fno-d
 # change __FILE__ to the relative path from the srctree
 KBUILD_CFLAGS	+= $(call cc-option,-fmacro-prefix-map=$(srctree)/=)
 
-KBUILD_CFLAGS	+= -g
+#KBUILD_CFLAGS	+= -g
 # $(KBUILD_AFLAGS) sets -g, which causes gcc to pass a suitable -g<format>
 # option to the assembler.
-KBUILD_AFLAGS	+= -g
+#KBUILD_AFLAGS	+= -g
 
 # Report stack usage if supported
 # ARC tools based on GCC 7.1 has an issue with stack usage
